src/views/InvolvementProfile/components/Membership/components/MemberList/index.js

```diff
@@ -33,7 +33,19 @@ const MemberList = ({
     return () => window.removeEventListener('resize', resize);
   });
 
+  // compare names by participation Hierarchy "Advisor" > "Leader" >
+  //"member" > "Guest" then alphabetically.
+
   const compareByLastThenFirst = (a, b) => {
+    if (a.ParticipationDescription !== b.ParticipationDescription) {
+      if (a.ParticipationDescription === 'Advisor') return -1;
+      if (b.ParticipationDescription === 'Advisor') return 1;
+      if (a.ParticipationDescription === 'Leader') return -1;
+      if (b.ParticipationDescription === 'Leader') return 1;
+      if (a.ParticipationDescription === 'Member') return -1;
+      if (b.ParticipationDescription === 'Member') return 1;
```

Comment:
I have never seen a function like this, where every line is a return statement- is there a reason you can't use 
```suggestion
      if (
        (a.ParticipationDescription === 'Advisor') ||
        (a.ParticipationDescription === 'Leader') ||
        (a.ParticipationDescription === 'Member'))
          return -1;
      if (
        (b.ParticipationDescription === 'Advisor') ||
        (b.ParticipationDescription === 'Leader') ||
        (b.ParticipationDescription === 'Member'))
          return 1;
    }
```
?

Reply:
This was the better way of improving the hierarchy of the group page by still using the the `compareByLastTheFirst` function. @EjPlatzer inspired me on this one.

Reply:
To clarify, we couldn't use the way you recommended @josh-peters-2357 because the order is important. Your way would return `-1` is a is a Member and b is an advisor, which is not at all the desired behavior. The way this is written, it will return the appropriate value based on which one has the "higher" role.